src/messenger-shortcuts.js

```diff
@@ -38,15 +37,43 @@ var HELP_TEXT =
 "<b>Alt+Shift+" + KEYS.HELP + "</b> &ndash; Display this help dialog<br>";
 
 
+let searchByTexts = {
+  search_conversation: "Search in Conversation",
+  send_a_like: "Send a Like",
+  new_message: "New Message",
+  conversation_information: "Conversation Information",
+  conversations: "Conversations",
+  conversation_actions: "Conversation actions"
+};
+
+
 /** Global variables and listeners **/
 
 // Tracks whether the like button is down
 var likeDown = false;
 
+/**
+ * Load the language of the messenger window
+ */
+window.addEventListener('load', function () {
+  let lang = document.documentElement.lang
+  if(lang !== undefined && lang !== null) {
+
+    const url = chrome.runtime.getURL('lang/' + lang + '.json');
+
+    fetch(url)
+        .then((response) => {
```

Comment:
What happens if the file for a language doesn't exist? Can we display some kind of meaningful error?

Reply:
If the file does not exists, then there is default strings we are searching by -> see [line 40](https://github.com/guoguo12/messenger-shortcuts/pull/25/files#diff-887621f197f6d2c765c3d38ee762f5f3b05edc07ec6b9ed9b08719b50ff13f28R40). (I've added catch to prevent uncaught exception)